src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -340,6 +340,12 @@ class ProductConfigFormController {
       if (includes(error.data, 'already in the cart')) {
         this.errorAlreadyInCart = true
         this.onStateChange({ state: 'errorAlreadyInCart' })
+      } else if (error.data && error.data.messages && error.data.messages[0] && error.data.messages[0].id === 'field.invalid.decimal.format') {
```

Comment:
As you're making `amount` apart of  `this.itemConfigForm` on the PR https://github.com/CruGlobal/give-web/pull/1068.  I don't think it will make it down to this part as the function would have returned on line 310.

Reply:
Yeah, it would be hard to get down here, it is more of a failsafe. What I did notice when testing just now is that, if it returns on line 310, it gives no feedback to the user. I'll see if I can remedy that as part of this PR.